StylaSEO/Controllers/Frontend/StylaApi.php

```diff
@@ -143,13 +143,22 @@ public function productsAction()
 
         $res = array();
         foreach ($result['data'] as $key => $value) {
-            $imgRes = $articles->getArticleListingCover($value['id']);
+            $mainImg = $articles->getArticleListingCover($value['id']);
+            $additionalImages = $articles->sGetArticlePictures($value['id'], false, 0, null, true);

```

Comment:
`sGetArticlePictures` seems like a typo to me, or is it a fancy name for a special method?

Reply:
@grzegorzstyla it's the name of a Shopware native method:
https://github.com/shopware/shopware/blob/5.3/engine/Shopware/Core/sArticles.php#L1412